exonum/src/blockchain/block.rs

```diff
@@ -60,6 +107,49 @@ pub struct Block {
     /// These calls can include transactions, `before_transactions` and/or `after_transactions` hooks
     /// for services.
     pub error_hash: Hash,
+    /// Some additional entries that can be added into the block.
+    pub entries: BlockHeaderEntries,
```

Comment:
Hmm... Maybe it makes sense to include the `exonum` version here as well, so the client code will be able to make assumptions about the contents of the `entry` section?
This will also fit with the documentation: for every optional field we will be able to provide description like "Added in `exonum 1.2.0`" or "Removed in `exonum 1.5.7`".
On the other hand, it's a constant overhead in storage. Maybe we can just create an index with mapping ("version" -> "height").

@slowli , what do you think?
cc @alekseysidorov